tests/rpc/test_rpc_telegram.py

```diff
@@ -1208,15 +1208,6 @@ def test_forcesell_handle_invalid(default_conf, update, mocker) -> None:
     assert msg_mock.call_count == 1
     assert 'not running' in msg_mock.call_args_list[0][0][0]
 
-    # No argument
```

Comment:
I'd  appreciate a quick test on this...

maybe look at `test_forceenter_no_pair` for an example - while it doesn't test the "full flow" - it can still test most parts of it - at least making sure there's no broken code (in the future ...)